modules/n1ql/pages/n1ql-language-reference/searchfun.adoc

```diff
@@ -300,6 +300,36 @@ This query returns 10 results, and the results are ordered, as specified by the
 As an alternative, you could limit the number of results and order them using the N1QL xref:n1ql-language-reference/limit.adoc[LIMIT] and xref:n1ql-language-reference/orderby.adoc[ORDER BY] clauses.
 ====
 
+.Search against an FTS index that carries a custom type mapping
+====
+[source,n1ql]
+----
+SELECT META(t1).id
+FROM `travel-sample` AS t1
+WHERE t1.type = "airline" AND SEARCH(t1.country, "United States");
```

Comment:
I'm not sure what this example is supposed to be demonstrating. Is the SEARCH function here meant to have an index hint with a custom mapping?

Reply:
The explicit t.type = "airline" becomes necessary when an index with a custom mapping is used.

Reply:
Resolved by #1105